packages/nimble-blazor/Examples/Demo.Shared/Pages/ComponentsDemo.razor

```diff
@@ -228,6 +228,22 @@
                 <NimbleListOption Value="tab-3">Tab 3</NimbleListOption>
             </NimbleSelect>
         </div>
+        <div class="sub-container">
+            <div class="container-label">Tabs - Anchor</div>
+            <NimbleAnchorTabs @bind-ActiveId="@ActiveAnchorTabId">
+                <NimbleAnchorTab id="tab-1">Tab 1</NimbleAnchorTab>
+                <NimbleAnchorTab id="tab-2">Tab 2</NimbleAnchorTab>
+                <NimbleAnchorTab id="tab-3" Disabled="true">Tab 3 (Disabled)</NimbleAnchorTab>
+                <NimbleTabsToolbar>
+                    <NimbleButton>Toolbar Button</NimbleButton>
+                </NimbleTabsToolbar>
+            </NimbleAnchorTabs>
+            <NimbleSelect @bind-Value="@ActiveAnchorTabId">
+                <NimbleListOption Value="tab-1">Tab 1</NimbleListOption>
+                <NimbleListOption Value="tab-2">Tab 2</NimbleListOption>
+                <NimbleListOption Value="tab-3">Tab 3</NimbleListOption>
+            </NimbleSelect>
+        </div>
```

Comment:
Our goal is to keep the Angular and Blazor examples in-sync. Based on that:
- Can you add these selects for changing the activeid to the Angular example
- Can you add a label to the select so we know that it does

Reply:
Done